backuprestore.adoc

```diff
@@ -800,12 +797,14 @@ value.
 
 Three types of restore procedures can be run:
 
-The `zmrestore` command is used to restore the mailboxes while the
+* The `zmrestore` command is used to restore the mailboxes while the
 {product-name} mailbox server is running.
 
-* `zmrestoreoffline` -- to restore the mail server when the mail server is
-down. This command is run for disaster recovery.
-* `zmrestoreldap` -- to restore the content of the LDAP directory server.
+* The `zmrestoreoffline` command is used to restore the mailbox server when
+it is stopped. This command is run for disaster recovery.
```

Comment:
Suggest:
The `zmrestoreoffline` command is used to restore the mailbox server when
+just the mailboxd process is stopped. This command is run for disaster recovery.